docs/GettingStarted/configuration/ExporterWebhook.md

```diff
@@ -0,0 +1,203 @@
+# Webhook Exporter
+
+The Webhook exporter is capable of receiving data from any service that sends an appropriate HTTP POST request.
+
+The HTTP POST request consists of two parts, [Header and the Payload](#webhook-headers-and-payloads). The Header data specifies which webhook plugin should be used to validate and transform the payload into a Pelorus metric, that is consumed by Prometheus.
+
+The payload data received by the webhook exporter may be any type of data that other exporters has to pull from other providers or OCP objects, such as [Failure](./ExporterFailure.md), [Commit Time](./ExporterCommittime.md), [Deploy Time](./ExporterDeploytime.md) and more.
```

Comment:
I would rephrase here. something like "The payload data received by the webhook exporter may be of the following types: <link to the types under payload-specification section>"

Reply:
We may and will have other type of metrics (SLA/SLO as an example).